src/UI/templates/default/Glyph/tpl.glyph.html

```diff
@@ -0,0 +1,80 @@
+<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted -->" <!-- BEGIN with_action -->href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}">
+
+
+<!-- BEGIN settings -->
+<span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+<!-- END settings -->
+
+<!-- BEGIN collapse -->
+<span class="glyphicon glyphicon-triangle-bottom" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END collapse -->
+
+<!-- BEGIN expand -->
+<span class="glyphicon glyphicon-triangle-right" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END expand -->
+
+<!-- BEGIN add -->
+<span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>
+<!-- END add -->
+
+<!-- BEGIN remove -->
+<span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>
```

Comment:
I suggest to use "glyphicon glyphicon-minus" here, which is already used for add/remove contexts in ILIAS (e.g. in T&A, user component etc.).